chapter03/Task04.py

```diff
@@ -7,17 +8,26 @@
       '"=" - Если я ответил верно')
 
 number = 50
+a = 0
+b = 101
+con = 0
 
 ans = str()
 while True:
```

Comment:
This cycle could become infinite if user is misbehaving or program has a bug. Consider stopping the program in that case with some message that something is going wrong either with user or with the program.